docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -4785,7 +4785,6 @@ image](#create-an-image) section for more details.
     Content-Type: application/json
 
     {
-      "Name": "top",
```

Comment:
No, the model is that the client should send the entire spec on a update.

See https://github.com/docker/swarmkit/issues/1394#issuecomment-240850719